Week3/homework/addToShoppingCart.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+function addToShoppingCart(item) {
+    // An array with two Items
+    var array = ['bananas', 'milk'];
+
+    // Push an item from the prameter
+    array.push(item);
+    // I check the length of the array if that greater than 1 the number of the array begins from 0
+    if (array.length >= 1) {
+        // With shift method i can remove the first item from the array
+        array.shift()
+    }
+    return `You bought ${array} !`;
+
+}
+console.log(addToShoppingCart('Tomato')); //Output You bought milk,Tomato !
+console.log(addToShoppingCart('Water')); //You bought milk,Water ! 
+console.log(addToShoppingCart('Orange')); //You bought milk,Orange !
```

Comment:
The question is asking that you can hold maximum 3 items in the shopping cart, so the expected output should be:
```
You bought  bananas,milk,Tomato !
You bought  milk,Tomato,Water !
You bought  Tomato,Water,Orange !
```
See if you can update this code to fix this. Hint: Update the check of the array length!

Reply:
Thank you for the feed back I fixed